.ci/eslint-plugin-zotero-translator/index.js

```diff
@@ -6,68 +6,93 @@
 
 const fs = require('fs');
 const path = require('path');
+const child_process = require('child_process');
+
+const repo = path.resolve(path.join(__dirname, '..', '..'));
+function exec(cmd, cwd=null) {
+	return child_process.execSync(cmd, { cwd: cwd || repo, encoding: 'utf8' });
+}
 
 const header_var_name = '__eslint_zotero_translator_header';
 
 const headers = {};
 const deleted = new Set(
-	fs.readFileSync('deleted.txt', 'utf-8')
+	fs.readFileSync(path.join(repo, 'deleted.txt'), 'utf-8')
 		.split('\n')
 		.map(line => line.split(' ')[0])
 		.filter(id => id && id.indexOf('-') > 0)
 );
 
+const branch = exec('git rev-parse --abbrev-ref HEAD').trim();
+// branch to compare lastUpdated against -- assume that if you're not in CI, you have upstream/master
```

Comment:
That's not a safe assumption. A few people have write access to zotero/translators, and they might have it as origin/master. Would be better to check the actual URL from `git remote -v`.

Reply:
So would it be OK to test `git remote -v` for `upstream/master` and assume `master` if it's not present? That would test against `master` for people who have `origin/master` access.

Reply:
Yeah, I think that'd be fine.

Reply:
Done.